cartridges/int_alma/cartridge/scripts/helpers/almaPaymentHelper.js

```diff
@@ -297,7 +297,7 @@ function buildPaymentData(installmentsCount, deferredDays, locale) {
             ipn_callback_url: URLUtils.http('Alma-IPN').toString(),
             customer_cancel_url: URLUtils.https('Alma-CustomerCancel').toString(),
             locale: locale,
-            origin: 'online',
+            origin: 'online_in_page',
```

Comment:
Is well only for InPage ? There are not condition to send 'online' if is not InPage ?